frontend/index.html

```diff
@@ -26,19 +26,6 @@
       window.APP_CONFIG = JSON.parse(
         '{"root": "/account/", "graphqlEndpoint": "/graphql"}',
       );
-      (function () {
-        const query = window.matchMedia("(prefers-color-scheme: dark)");
-        function handleChange(list) {
-          if (list.matches) {
-            document.documentElement.classList.add("cpd-theme-dark");
-          } else {
-            document.documentElement.classList.remove("cpd-theme-dark");
-          }
-        }
-
-        query.addEventListener("change", handleChange);
-        handleChange(query);
-      })();
```

Comment:
do we not support dark colour themes anymore? Or is this handled some other way?

Reply:
We do, it's me who misunderstood how the design tokens worked at some point. It defaults to media queries, but you can override the theme by setting the `cpd-theme-dark` or `cpd-theme-light` classes.

So this piece of code isn't needed